ZKSyncSDK/Classes/Model/TimeRange.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  TimeRange.swift
+//  ZKSync
+//
+//  Created by Eugene Belyakov on 20/03/2021.
+//
+
+import Foundation
+
+public struct TimeRange: Decodable {
+    public var validFrom: UInt64
+    public var validUntil: UInt64
+    
+    public init(validFrom: UInt64, validUntil: UInt64) {
```

Comment:
There should be the ability to set the maximum possible time range by default

Reply:
Done